iocore/eventsystem/UnixEThread.cc

```diff
@@ -228,8 +224,9 @@ EThread::execute_regular()
     do {
       done_one = false;
       // execute all the eligible internal events
-      EventQueue.check_ready(cur_time, this);
-      while ((e = EventQueue.dequeue_ready(cur_time))) {
+      ink_hrtime event_time = Thread::get_hrtime_updated();
```

Comment:
I don't think we should get the updated time here, but should use the cached time from earlier in the loop.

Reply:
fixed.